roof/server.js

```diff
@@ -101,12 +86,12 @@ roofMotor.setStateCallback(function(state) {
         case "ERROR": break;
     }
 
-    roofState = state;
+    physicalState = state;
 });
 
 function batteryReporter() {
-    var status = roofMotor.status();
-    var data = {
+    let status = roofMotor.status();
+    let data = {
```

Comment:
const > let > var, when applicable. I think most of the local variables could be consts.